tokio/Cargo.toml

```diff
@@ -111,6 +111,7 @@ tracing = { version = "0.1.29", default-features = false, features = ["std"], op
 # Requires `--cfg tokio_unstable` to enable.
 [target.'cfg(all(tokio_unstable, target_os = "linux"))'.dependencies]
 io-uring = { version = "0.7.6", default-features = false, optional = true }
+linux-raw-sys = { version = "0.12.1", optional = true }
```

Comment:
Is there a particular reason to not use `libc` instead ?
AFAIS it provides all the constants used in this PR from `linux-raw-sys`